server.go

```diff
@@ -111,9 +111,9 @@ func NewServer() *Server {
 		MetricInterval:      0,
 		DiagnosticInterval:  0,
 
-		LogOutput: os.Stderr,
+		LogOutput: ioutil.Discard,
```

Comment:
can we remove LogOutput entirely now?

Reply:
I left that there because `func (m *Command) Close()` uses it to close the writer. I didn't want to make `Close()` part of the `Logger` interface, but I'm not clear on the best way to close that writer.

Reply:
I'm not sure why that's there - it wouldn't be used normally since Stdout/Stderr don't implement io.Closer (afaik). What we could do is see if Server.Logger is an io.Closer and if so, call Close on it, though I'd prefer to remove it entirely. @yuce Do you know why Command.Close tries to call Close on Server.LogOutput?

```go
// Close shuts down the server.
func (m *Command) Close() error {
	var logErr error
	serveErr := m.Server.Close()
	logOutput := m.Server.LogOutput
	if closer, ok := logOutput.(io.Closer); ok {
		logErr = closer.Close()
	}
...
```

Reply:
That makes sense to me, and I just tested it with an implementation of `Logger` that also implements `io.Closer()` and that works as expected. So the idea being if someone needed their logger to close, it would get called here. I'll push a commit for that change.

Reply:
I made that change in ff25e2e01bb7c17d71b7d9dff6b68d1ccd02f77a